README.md

```diff
@@ -31,76 +31,39 @@
-    |---|---|---|
-    | `AWS_ACCOUNT_NAME` | This should match your AWS account name or the account ID. | `re-managed-observe-production` |
-    | `AWS_REGION` | Should represent AWS region. Stick to London. | `eu-west-2` |
-    | `AWS_DEFAULT_REGION` | Default AWS region. | `eu-west-2` |
-    | `CLUSTER_NAME` | The name of the cluster about to be created. Needs to be unique across your entire Hosted Zone. | `cluster1` |
-    | `ZONE_ID` | An AWS Hosted Zone ID which you've obtained from the first step of this guide. | `Z00000000000` |
-    | `ZONE_NAME` | An AWS Hosted Zone name which you've obtained from the first step of this guide. | `re-managed-observe-production.aws.ext.govsvc.uk` |
-
-    We've prepared a [templater script](https://github.com/alphagov/gsp-teams/blob/master/scripts/create_cluster_config.sh) to create the new cluster terraform declaration based on a [terraform template](https://github.com/alphagov/gsp-teams/blob/master/terraform/templates/cluster.tf)
-
-    With the above variables you can run:
+    This leaves you with a manual steps of:
 
     ```sh
-    aws-vault exec run-sandbox -- ./scripts/create_cluster_config.sh
-    ```
+    cd terraform/clusters/${DOMAIN}
 
-    This should generate new file at the location:
+    aws-vault exec run-sandbox -- terraform init -upgrade=true
 
-    ```
-    terraform/clusters/${CLUSTER_NAME}.${AWS_ACCOUNT_NAME}.aws.ext.govsvc.uk/cluster.tf
+    aws-vault exec run-sandbox -- terraform apply
     ```
 
-    This leaves you with a manual steps of:
+1. Bootstrap cluster
+
+    Clone the [`gsp-terraform-ignition`](https://github.com/alphagov/gsp-terraform-ignition) repo:
 
     ```sh
-    export DOMAIN=${CLUSTER_NAME}.${AWS_ACCOUNT_NAME}.aws.ext.govsvc.uk
-    cd terraform/clusters/${DOMAIN}
+    cd bootstraper/
 
-    # initialise terraform
-    aws-vault exec run-sandbox -- docker run -it \
-      --env AWS_DEFAULT_REGION \
-      --env AWS_REGION \
-      --env AWS_ACCESS_KEY_ID \
-      --env AWS_SECRET_ACCESS_KEY \
-      --env AWS_SESSION_TOKEN \
-      --env AWS_SECURITY_TOKEN \
-      --env DOMAIN \
-      --volume=$(pwd)/../../../:/terraform \
-      -w /terraform/terraform/clusters/${DOMAIN} \
-      govsvc/terraform init
-
-    # apply the plan
-    aws-vault exec run-sandbox -- docker run -it \
-      --env AWS_DEFAULT_REGION \
-      --env AWS_REGION \
-      --env AWS_ACCESS_KEY_ID \
-      --env AWS_SECRET_ACCESS_KEY \
-      --env AWS_SESSION_TOKEN \
-      --env AWS_SECURITY_TOKEN \
-      --env DOMAIN \
-      --volume=$(pwd)/../../../:/terraform \
-      -w /terraform/terraform/clusters/${DOMAIN} \
-      govsvc/terraform apply
+    ./bootstrap.sh run-sandbox
     ```
 
-1. Test the connection to Kubernetes by executing the following:
-    ```
-    cp $(pwd)/bootkube-assets/auth/user-config ./kubeconfig
-    export KUBECONFIG="$(pwd)/kubeconfig"
-    aws-vault exec run-sandbox -- kubectl get all --all-namespaces
+1. Destroy bootstrap node
```

Comment:
This step is not necessary if you're using the `bootstrap.sh` script as specified in the previous step...it does this for you.